### PR TITLE
Add brakeman

### LINKED ISSUE #716: Brakeman
We would like to install the [brakeman gem](https://github.com/presidentbeef/brakeman) to help with security.

- [ ] Install and configure brakeman gem
- [ ] Run initial scan
- [ ] Correct or file issues for any warning found in the scan

### PR DESCRIPTION
**Pull request by sinn22**
# Checklist:

- [x] I have performed a self-review of my own code,
- [x] I have commented my code, particularly in hard-to-understand areas,
- [x] I have made corresponding changes to the documentation,
- [x] I have added tests that prove my fix is effective or that my feature works,
- [x] New and existing unit tests pass locally with my changes ("bundle exec rake"),
- [x] Title include "WIP" if work is in progress.

Resolves #716 and #717 

### Description

Add brakeman gem and run brakeman in CI

### Type of change

* New feature (non-breaking change which adds functionality)
* This change requires a documentation update

### CODE REVIEW

File: app/controllers/locations_controller.rb [hunk 1]
```diff
@@ -42,7 +42,7 @@ def destroy
   private
 
   def set_location
-    @location = Location.find(params[:id])
+    @location = @facility.locations.find(params[:id])
```
Comment (sinn22) [inline app/controllers/locations_controller.rb:45]: fix this brakeman Pointed out
```
Confidence: Weak
Category: Unscoped Find
Check: UnscopedFind
Message: Unscoped call to `Location#find`
Code: Location.find(params[:id])
File: app/controllers/locations_controller.rb
Line: 45
```

File: app/controllers/file_uploads_controller.rb [hunk 2]
```diff
@@ -25,7 +25,7 @@ def new
 
   def show
     @processed_file = ProcessedFile.find(params[:id])
-    record_class = @processed_file.category.constantize
+    record_class = CsvImporter::CATEGORIES.find{ |v| v === @processed_file.category}.constantize
```
Comment (sinn22) [inline app/controllers/file_uploads_controller.rb:28]: fix this brakeman pointed out
```
Confidence: Medium
Category: Remote Code Execution
Check: UnsafeReflection
Message: Unsafe reflection method `constantize` called with model attribute
Code: ProcessedFile.find(params[:id]).category.constantize
File: app/controllers/file_uploads_controller.rb
Line: 28
```

**Comment by sinn22** [comment]
@CraigJZ 
I fixed all the pointed out by brakeman. And add brakeman for Makefile and README.
Please, run workflows and check my file changes!

**Comment by sinn22** [comment]
I'm sorry, I fixed the falling lint.

**Review by CraigJZ — APPROVED** [review]
@sinn22 This looks good. Thank you for updating the docker-compose and Makefile too!

**Pull request merged**